src/Alpha_complex/concept/SimplicialComplexForAlpha.h

```diff
@@ -23,7 +23,7 @@ struct SimplicialComplexForAlpha {
   typedef unspecified Simplex_handle;
   /** Handle to specify a vertex. Must be a non-negative integer. */
   typedef unspecified Vertex_handle;
-  /** Handle to specify the simplex filtration value. */
+  /** Handle to specify the simplex filtration value. Must be capable to represent the special value "Not-A-Number"*/
```

Comment:
Unrelated, but we should drop "handle" from this line, it looks like a bad copy-paste from the 2 previous lines.